prepicking_pose:  
  position: 
    x: 0.1
    y: -0.5
    z: 0.6
  orientation: 
    x: 0.9264822696150599
    y: -0.37589781415696555
    z: -0.013352019723230064
    w: 0.012375822003138056

picking_pose:
  position: 
    x: 0.1
    y: -0.5
    z: 0.13
  orientation: 
    x: 0.9264760972024011
    y: -0.37591251000400594
    z: -0.013375082523185242
    w: 0.012366620500083948

preplace_pose:
  position: 
    x: 0.1
    y: 0.5
    z: 0.6
  orientation: 
    x: 0.9240724689450894
    y: -0.38221696214191697
    z: -0.0002749884454337605
    w: 0.00043631420553596426

place_pose:
  position: 
    x: 0.1
    y: 0.5
    z: 0.13
  orientation: 
    x: -0.9242633024642829
    y: 0.38175513154697743
    z: -0.0006031891023182969
    w: 5.8466008778389534e-05
